modules/ROOT/pages/exchange-api.adoc

```diff
@@ -259,6 +260,50 @@ https://anypoint.mulesoft.com/exchange/api/v2/assets/:groupId/:assetId/:version/
   }'
 ----
 
+[[update-mutable-data]]
+=== Update Mutable Data
+
+This example shows how to update the mutable data of an existing asset and describe the asset in the same request. The mutable data of an asset includes categories, custom fields, tags, and documentation pages.
+
+This example publishes the category and custom field, which have the keys `categoryKey` and `fieldKey`. Publication fails if the category or the custom field is not configured for your organization.
+
+This example updates the asset's documentation by uploading a documentation file named `files.docs.zip`. To extend this documentation, include each documentation page as a Markdown file in the root directory of the archive with a name such as `Getting Started.md`. Replace `Getting Started` with the title of the page. To include a terms and conditions page, put the content in a file named `terms.md`. Include resources such as images in the optional subdirectory `resources`. Specify the page order with the property `pageOrder` on the root of the main object in the optional file `config.json`:
```

Comment:
What is the difference between updating and extending the documentation?

Reply:
This paragraph reads more like task steps than example introduction.

Reply:
Rewrote to clarify the documentation file structure, and to describe the example and not list task steps.